src/cloud-cli/userdb.ts

```diff
@@ -133,6 +135,45 @@ export async function getUserDb(host: string, port: string, dbName: string) {
   }
 }
 
+export function migrate() {
+  const logger = createGlobalLogger();
+
+  // read the yaml file
+  const configFile: ConfigFile | undefined = loadConfigFile(dbosConfigFilePath);
+  if (!configFile) {
+    logger.error(`failed to parse ${dbosConfigFilePath}`);
+    return;
+  }
+
+  let dbType = configFile.database.user_dbclient;
+  if (dbType == undefined) {
+    dbType = 'knex';
+  }
+
+  const migratecommands = configFile.database.migrate;
+  const rollbackcommands = configFile.database.rollback;
+  
+  try {
+
+    migratecommands?.forEach((cmd) => {
+        const command = "npx "+ dbType + " " + cmd; 
+        logger.info("Executing " + command);
+        execSync(command);
+    })
+  } catch(e) {
+    const err: Error = e as Error;
+    logger.error(err);
+
+    rollbackcommands?.forEach((cmd) => {
+      const command = "npx " + dbType + " " + cmd; 
+        logger.info("Executing " + command);
+        execSync(command);
+    })
+
+  }
+
+}
```

Comment:
I think there is a use case for users wanting to trigger a rollback manually.

Reply:
I can agree with that. Let us what @kraftp thinks. In we do that should add a npx db-cloud userdb rollback